sidecar/supervisor/appsupervisor.go

```diff
@@ -23,6 +23,9 @@ import (
 
 	"time"
 
+	"sync"
+
+	"github.com/Shopify/toxiproxy/Godeps/_workspace/src/github.com/Sirupsen/logrus"
```

Comment:
This dependency needs to be fixed. It was probably imported because `logrus` is renamed just `log` in this file.